packages/amplify_auth_cognito/example/ios/unit_tests/amplify_auth_cognito_tests.swift

```diff
@@ -348,6 +348,30 @@ class amplify_auth_cognito_tests: XCTestCase {
             }
         })
     }
+
+    func test_confirmSignUpForwardOptions() {
+        let mockOptions: Dictionary<String, Any> = ["clientMetadata": ["key": "value"]]
```

Comment:
Nit: Maybe we could extract to a `mockClientMetadata` to reuse on the assert

Reply:
This is left as this on purpose... the assertion is happening inside the mock class `ConfirmSignUpMock`, the assertion function doesn't have access to variables declared outside of this mock class. So I just used the literal inside the mock class and without reusing the mocked variable.